controllers/pravega_controller.go

```diff
@@ -403,7 +403,13 @@ func MakeControllerService(p *api.PravegaCluster) *corev1.Service {
 }
 
 func MakeControllerPodDisruptionBudget(p *api.PravegaCluster) *policyv1beta1.PodDisruptionBudget {
-	minAvailable := intstr.FromInt(int(p.Spec.Pravega.MaxUnavailableControllerReplicas))
+	var maxUnavailable intstr.IntOrString
+
+	if p.Spec.Pravega.MaxUnavailableControllerReplicas == int32(1) {
```

Comment:
Did you intend to use another parameter here?
The description says the decision is to be based on replica count:
> Also if the replica count is 1 setting maxunavailable replicas to 0 as in segment store

Reply:
Thanks @jkhalack . I have corrected it.